server/common/pom.xml

```diff
@@ -43,6 +43,22 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
+    <name>Broadcom COBOL LS Common Module</name>
+    <description>Provides dialect's support for 3rd party companies</description>
+    <url>https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support</url>
+
+    <licenses>
```

Comment:
@zimlu02 please check if Apache License, Version 2.0 fits to our needs

Reply:
But we are EPL2 https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/LICENSE.md
Why do you want to publish it under apache?

Reply:
Fixed.